.env-dist

```diff
@@ -172,3 +172,10 @@ MAX_MANUAL_SCANS=100
 MAX_INITIAL_SCANS=100
 MAX_PROFILES_ACTIVATED=100
 MAX_PROFILES_CREATED=100
+
+# Footer links
+
+NEXT_PUBLIC_MONITOR_FAQ=https://support.mozilla.org/kb/firefox-monitor-faq
+NEXT_PUBLIC_MONITOR_LEGAL=https://www.mozilla.org/about/legal/terms/subscription-services/
+NEXT_PUBLIC_MONITOR_SUBSCRIPTION_SERVICES=https://www.mozilla.org/privacy/subscription-services/
+NEXT_PUBLIC_MONITOR_GITHUB=https://github.com/mozilla/blurts-server
```

Comment:
After this and your other PR is merged, might be worth sending a message in the eng channel for people to update their `.env`s :)